diff --git a/lib/python/Screens/About.py b/lib/python/Screens/About.py
index d87973997..ca4cbe27d 100644
--- a/lib/python/Screens/About.py
+++ b/lib/python/Screens/About.py
@@ -8,6 +8,7 @@ from Components.NimManager import nimmanager
 from Components.About import about
 from Components.ScrollLabel import ScrollLabel
 from Components.Button import Button
+from Components.HdmiCec import HdmiCec
 
 from Components.Label import Label
 from Components.ProgressBar import ProgressBar
