diff -Naur XStreamity-master/XStreamity/usr/lib/enigma2/python/Components/Renderer/xRunningText.py XStreamity-PersianPrince/XStreamity/usr/lib/enigma2/python/Components/Renderer/xRunningText.py
--- XStreamity-master/XStreamity/usr/lib/enigma2/python/Components/Renderer/xRunningText.py	2020-06-20 04:02:50.000000000 +0430
+++ XStreamity-PersianPrince/XStreamity/usr/lib/enigma2/python/Components/Renderer/xRunningText.py	2020-06-20 21:13:33.476342504 +0430
@@ -35,7 +35,7 @@
 from Renderer import Renderer
 from skin import parseColor, parseFont
 
-import Plugins.Extensions.XStreamity.owibranding as xowibranding
+#import Plugins.Extensions.XStreamity.owibranding as xowibranding
 
 
 # scroll type:
@@ -88,27 +88,27 @@
 		self.mTimer = eTimer()
 		
 
-		try:
+#		try:
 			#from boxbranding import getImageDistro, getImageVersion, getOEVersion
-			self.mTimer.callback.append(self.movingLoop)
-		except:
-			try:
-				if xowibranding.getMachineBrand() == "Dream Multimedia" or xowibranding.getOEVersion() == "OE 2.2":
-					self.mTimer_conn = self.mTimer.timeout.connect(self.movingLoop)
-			except:
-				pass
+		self.mTimer.callback.append(self.movingLoop)
+#		except:
+#			try:
+#				if xowibranding.getMachineBrand() == "Dream Multimedia" or xowibranding.getOEVersion() == "OE 2.2":
+#					self.mTimer_conn = self.mTimer.timeout.connect(self.movingLoop)
+#			except:
+#				pass
 				
 		
 
 	def preWidgetRemove(self, instance):
 		self.mTimer.stop()
 		
-		try:
+#		try:
 			#from boxbranding import getImageDistro, getImageVersion, getOEVersion
-			self.mTimer.callback.remove(self.movingLoop)
-		except:
-			if getMachineBrand() == "Dream Multimedia" or getOEVersion() == "OE 2.2":
-				self.mTimer_conn = self.mTimer.timeout.disconnect(self.movingLoop)
+		self.mTimer.callback.remove(self.movingLoop)
+#		except:
+#			if getMachineBrand() == "Dream Multimedia" or getOEVersion() == "OE 2.2":
+#				self.mTimer_conn = self.mTimer.timeout.disconnect(self.movingLoop)
 				
 		
 		self.mTimer = None
